experiments/Marmot/Venn/tikzlibraryvenn.code.tex

```diff
@@ -0,0 +1,174 @@
+% Copyright 2019 by an anonymous marmot
```

Comment:
I wanted to merge the `venn` library, but the code is lacking in some places.  Would it also be possible to make it a PGF library rather than a pure Ti*k*Z library?